Commands.adoc

```diff
@@ -0,0 +1,226 @@
+= Available Commands and Examples
+:toc:
+:toc-placement!:
+
+Once you do `vagrant up` and run an ADB box, as a developer you may want to
+customize or set up your development environment to optimally utilize the ADB as
+ per your requirements.
+
+The vagrant-service-manager plugin enables you to check and tweak your
+container development environment.
+
+'''
+toc::[]
+'''
+
+You can use the vagrant service manager to set up your environment variables and
+ get the TLS certificates; identify the routable ip network and version of your
+ box; and check the status and manage the lifecycle of the configured services
+ in the running VM.
+
+To run the vagrant-service-manager plugin use: +
+-------------------------------------------
+vagrant service-manager [command] [options]
+-------------------------------------------
+The following commands can be used: +
+`env`: Displays the connection information for services in the ADB box +
+`box`: Displays the box related information like version, release, IP etc +
+`status`: Lists services and their running state +
+`start`: Starts the given service in the box +
+`stop`: Stops the given service in the box +
+`restart`: Restarts the given service in the box +
+
+The following options can be used: +
+`--script-readable`: Displays information in a key=value format +
+`-h, --help, print this help`: Prints help for the specific command being run +
+
+== Get the Environment Variables and Certificates
```

Comment:
drop the 'the' - Get Environment Variables and Certificates

and why do you capitalize Environment, Variables and Certificates?